tox.ini

```diff
@@ -74,8 +74,11 @@ commands =
 [flake8]
 show-source = True
 max-line-length = 100
-ignore = E203,E231,W503
 exclude = .git,.tox,dist,*egg
 
+# Configure flake8-import-order
+application-import-names = noggin, noggin_messages
```

Comment:
You need to add `flake8-import-order` to the dev dependencies if you want to use this feature.

Reply:
Done.